docs/quick-tutorials/k8s-istio-authorization-policies.mdx

```diff
@@ -32,7 +32,6 @@ Before you start, you'll need a Kubernetes cluster. Having a cluster with a [CNI
 
 You can now install (or reinstall) Otterize in your cluster, and optionally connect to Otterize Cloud. Connecting to Cloud lets you:
 1. See what's happening visually in your browser, through the "access graph";
-2. Avoid using SPIRE (which can be installed with Otterize) for issuing certificates, as Otterize Cloud provides a certificate service.
```

Comment:
Why drop this? Don't you still need to install SPIRE to issue certificates if you're not connecting to SPIRE?

Reply:
You don't need SPIRE or certificates in this tutorial at all.